### DISCUSSION TITLE
Notifications for pull request review requested always showing "requested your review"

### DISCUSSION THREAD

**Discussion by wolfy1339** [community/community#59966] [Mobile]
### Select Topic Area

Bug

### Body

In the notifications panel, every time there is a review requested it shows as if it's always requesting a review from the logged in user even if the review request for a completely different user.

This happens to all repositories, whether the user has wrote access or not to the repo

**Comment by alcere** [comment]
Hi @wolfy1339, thanks for reporting the issue, we are sorry for the troubles.

I tried to reproduce the issue and I couldn't, could you please share a picture/video of the problem, please? That would be extremely helpful!

Thanks!

**Reply by wolfy1339** [reply]
I don't currently have a notification that is a PR with a requested review.
As soon as I have one I will share a screenshot

I'll share the version info, that can be of relevance:
- App version: v1.12.0 (Beta channel)
- OS SDK: 33
- Device: Google Pixel 7 Pro

**Reply by wolfy1339** [reply]
![Screenshot_20230703-175740.png](https://github.com/community/community/assets/4595477/83249004-7e0f-4a0d-8a48-1e0c990a1d56)

See the notification for the `hellomouse/random-testing repo`

**Reply by alcere** [reply]
Thanks for sharing the screenshot! Please, let me know if I understood the problem correctly:
- Being logged with an account `A`.
- Another repo user,` B`, requested a review from `C`.
- `A` receives a notifications indicating that `B` requested a review from `A`?

I am sorry for the cryptic description here!

**Reply by wolfy1339** [reply]
That is correct.

**Reply by alcere** [reply]
Thanks for the clarification, I will open an internal issue to investigate this. Thanks for the heads up!
